### PR TITLE
fix for Add/delete sample data to modules test cases for Cypress test suite on WZD

### PR DESCRIPTION
**Pull request by mauceballos**
|Related issue|
|---|
|closes (https://github.com/wazuh/wazuh-kibana-app/issues/4440) |

## Description

This PR fixes add/delete sample data, I only increase the timeout

` cy.get(getSelector(sample, SAMPLE_DATA), { timeout: 9000 }).should('have.text',buttonLabel)
    cy.get(toastLocatorTitle, { timeout: 8000 })`
    for
   `cy.get(getSelector(sample, SAMPLE_DATA), { timeout: 15000 }).should('have.text',buttonLabel)
    cy.get(toastLocatorTitle, { timeout: 15000 })`

### E2E tests

<details>

</details>

### CODE REVIEW

File: test/cypress/cypress/integration/step-definitions/settings/sample-data/add-remove-sample-data.when.js [hunk 1]
```diff
@@ -7,14 +7,14 @@ When('The user {} sample data for', (status,types) => {
   let titleStatus = 'added'
   let buttonLabel = 'Remove data'
   if(status != 'adds'){titleStatus =  'removed'; buttonLabel = 'Add data';}
-  cy.log(titleStatus)
-  cy.log(buttonLabel)
+  // cy.log(titleStatus)
```
Comment (Mayons95) [inline test/cypress/cypress/integration/step-definitions/settings/sample-data/add-remove-sample-data.when.js:10]: Please delete these commented log lines!

**Review by Mayons95 — CHANGES_REQUESTED** [review]
Review the commented code

File: test/cypress/cypress/integration/step-definitions/settings/sample-data/add-remove-sample-data.when.js (hunk 1 above)
Comment (mauceballos) [reply test/cypress/cypress/integration/step-definitions/settings/sample-data/add-remove-sample-data.when.js]: Done

**Review by Mayons95 — APPROVED** [review]

**Review by Machi3mfl — APPROVED** [review]
LGTM!

**Pull request merged**